### DISCUSSION TITLE
Holopin badge board is not getting updated on github readme file

### DISCUSSION THREAD

**Discussion by d-coder111** [community/community#69893] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Holopin badge board is not getting updated on github readme file. Any solution for the same?

**Comment by siddharth9300** [comment]
@d-coder111 Did you find any solution? I have the same issue.

**Reply by d-coder111** [reply]
No I am still facing the issue.

**Reply by d-coder111** [reply, marked as answer]
Hey @siddharth9300 ,
I have found a way to resolve this.
You can update ur holopin badge by using this link-   [![@username's Holopin board] (https://holopin.io/api/user/board?user=username)] 
   (https://holopin.io/@username)

Note: Replace username with ur holopin username.
Do let me know if you could update with this.

There's another way of purging the image using git bash but I found it easier this way.

**Comment by AnkurBansal18** [comment]
Hey @d-coder111, you need to go to your profile on Holopin. **Then go to inventory and add the badges**(you need to select them manually) you want to add to the board. Once clicked, you will see them on the board on your Holopin profile. 

Finally, add the below text as a link to your GitHub readme. Adding this will show the same board as your Holopin profile on your GitHub readme as well.

 An image of @yourgithubsusername Holopin badges, which is a link to view their full Holopin profile(https://holopin.me/username) (https://holopin.io/@username) 
 
 I hope it helps :)

**Reply by d-coder111** [reply]
Hey I have done all these steps.. My concern is that even after updating holopin badge board.. It's not getting updated in readme file of my GitHub account.

**Reply by AnkurBansal18** [reply]
Then I am not sure of what's going on with your readme. Are you sure you are providing the exact Holopin link in the readme ?

**Reply by d-coder111** [reply]
Yes I am providing the correct holopin link.

**Reply by miguelgarcia210** [reply]
@d-coder111  Visit Holopin.io and view your profile to view your board. Below you should see the embedded link necessary.
I was having the same issue and realized Holopin has changed their endpoint. I just copied their new embedded link and committed that to my README file. All is working as expected now.

**Reply by d-coder111** [reply]
Yes.. I have done that way only.. Now it's working completely fine.

**Comment by SUDEEP-M-SHETTY** [comment]
Maybe your badges are not listed in public user profile, in order to enable it
1. Go to your holopin profile
2. Then click on badges
3. Click on a particular badge
4. Turn on the Listed in public user profile
